release-team/role-handbooks/bug-triage/README.md

```diff
@@ -26,28 +26,25 @@ Bug Triage Handbook
     - [Reports [Thaw]](#Reports-Thaw)
 
 ## Summary
-Primarily, your job is to make sure that bugs which affect the release are dealt with in a timely fashion.  
-Bugs can appear either as GitHub Issues or Pull Requests (PRs).  
+The role of the bug triage team is to make sure that Pull Requests (PRs) and Github Issues which affect the release are dealt with in a timely fashion.
 
-Some common tasks include:
-- Polling for bugs (mainly issues with the label `kind/bug`)
-- Checking status of open issues / PRs
-- Interlocking with relevant assignees / owners / sig-leads of issues/PRs to get status
+Primary tasks include:
+- Listing the status of open issues / PRs
+- Communicating with relevant assignees / owners / sig-leads of issues/PRs to get status
 - Updating issues/PRs, clarifying situations, enabling next level decision making
-- Publishing summary reports 
+- Presenting summary reports at release meetings
 
-The role has been interchangeably called "Bug Triage" and "Issue Triage", but the general term "issue" can be conflated with the specific GitHub usage of `is:issue` artifact type.  It is important to consider both `is:issue` and `is:pr` GitHub artifacts when triaging bugs.
-
-Secondarily, you will be helping improve automation around tracking issues against the current milestone.
-
-How this works depends on where you are in the release cycle.  
-There are four relevant periods where your workload changes:
+There are four relevant periods where the workload changes:
```

Comment:
This might be wrong, especially for this release cycle but for the last one too. IMO:
* The Early Release took from day 1 up to to a mid release (~week 6)
* Code Freeze is Coming: from mid release to code freeze - this is when we started pinging inactive issues/PRs and then extended to all issues
* Code Freeze and Code Thaw: once code freeze starts to the end of the cycle.